kpTranslator/config.py

```diff
@@ -2,10 +2,15 @@
 import connexion
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
+from kpTranslator import encoder
 
 basedir = os.path.abspath(os.path.dirname(__file__))
+db_file = os.path.join(basedir, 'translator_kg.db')
```

Comment:
At some point you'll have to change this to a different kind of connection when you deploy it somewhere since sqllite is not suitable for production.

Reply:
I agree, I was thinking the same thing. For now, I'm using sqlite for convenience with the idea that we will switch to something better for production. I'll be interested in recommendations that you have.